DATA/production/configurations/2022/MayJunePilotBeam/apass1/selectSettings.sh

```diff
@@ -23,11 +23,83 @@ if [[ $RUNNUMBER -ge 519903 ]] && [[ $RUNNUMBER -le 520099 ]]; then
   COLLISIONCONTEXT=collisioncontext_Single_4b_2_2_2_noLR.root
 fi
 
-# B field update
-if [[ $RUNNUMBER -ge 519041 ]]; then
+# LHC22f: 520143 - 520473
+if [[ $RUNNUMBER -ge 520143 ]] && [[ $RUNNUMBER -le 520473 ]]; then
```

Comment:
What about runs [519041, 520099]? The B field here was positive, like this, negative will be taken from line 5 (unless I am overlooking something).